secretmanager/api/SecretManagerTest/SamplesTest.cs

```diff
@@ -30,6 +30,7 @@ namespace GoogleCloudSamples
     public class SampleTests : IClassFixture<SecretsFixture>
     {
         private static readonly string s_projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+        private static readonly string s_iamUser = $"user:sethvargo@google.com";
```

Comment:
Hmmm, don't think this should be public like this. Use an env variable that we can set on CI

Reply:
Can you please create a new service account in the project for me? Then we can update this to use that service account and pull its value from an environment variable.

Reply:
Please set the value of `DOTNET_SECRETMANAGER_SAMPLES_IAM_USER` to `serviceAccount:<email>`.